### PR TITLE
update troubleshooting.md docs

### PR DESCRIPTION
**Pull request by ArafatAbdussalam**
I fixed the grammatical errors on the docs

### CODE REVIEW

**Comment by minrk** [comment]
This PR now has merge conflicts, due to earlier contributions on the same file. If you resolve the conflicts, we can review this contribution.

If you choose to open a new PR, please close this issue with a link to the new PR.

**Comment by minrk** [comment]
@ArafatAbdussalam this PR still has some conflicts. If you can resolve those (I think the contributing/docs.rst changes are included by mistake?), we can review this PR.

If you choose to open a new PR, please close this issue with a link to the new PR.

If you don’t plan to finish this PR since the outreachy contribution period has ended, let us know and we will take over the remaining issues.

File: docs/source/troubleshooting.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ With a docker container, pass in the environment variable with the run command:
 Some certificate providers, i.e. Entrust, may provide you with a chained
 certificate that contains multiple files. If you are using a chained
 certificate you will need to concatenate the individual files by appending the
-chain cert and root cert to your host cert:
+chainined cert and root cert to your host cert:
```
Comment (consideRatio) [inline docs/source/troubleshooting.md:246]:
```suggestion
chained cert and root cert to your host cert:
```

**Review by consideRatio — APPROVED** [review]
LGTM besides the need to resolve merge conflicts and the code suggestion I added about the word "chained"

**Pull request merged**